packages/devtools-reps/src/object-inspector/component.js

```diff
@@ -212,6 +213,10 @@ class ObjectInspector extends Component<Props> {
         createObjectClient,
         createLongStringClient
       );
+
+      if (recordTelemetryEvent) {
+        recordTelemetryEvent("object_expanded");
+      }
```

Comment:
I think it would be nice to have a default prop here which would let you just do `recordTelemetryEvent`. There are obviously other strategies too with context or modules that would help too, so if you want to add the probes and then refactor, tht works too